### PR TITLE
util: Add tests for blob helpers

### PR DESCRIPTION
**Pull request by scorbajio**

### CODE REVIEW

File: packages/util/test/blobHelpers.spec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { assert, describe, it } from 'vitest'
+
+import { commitmentsToVersionedHashes, computeVersionedHash, getBlobs } from '../src/index.js'
+
+describe('YourClass', () => {
```
Comment (acolytec3) [inline packages/util/test/blobHelpers.spec.ts:5]: Total nit but can we rename this to something that reflects the test case, just `blob helper tests` at a minimum?

**Review by acolytec3 — CHANGES_REQUESTED** [review]
Looks great, with one edit on the test case name

File: packages/util/test/blobHelpers.spec.ts (hunk 1 above)
Comment (scorbajio) [reply packages/util/test/blobHelpers.spec.ts]: Fixed.

**Review by gabrocheleau — APPROVED** [review]

**Review by acolytec3 — APPROVED** [review]
Lgtm

**Pull request merged**